.rubocop.yml

```diff
@@ -16,9 +16,13 @@ Gemspec/OrderedDependencies:
 Lint/UnusedMethodArgument:
   Enabled: false
 
+# We have a couple of empty methods especially when we implement an API contract on our Action subclasses
+# and those look nicer to us when still written as expended (in separate lines) to suggest us to fill them at some point
+# rather than the compact representation of having them one-liner `def foo; end` that is rubocop's default.
 Style/EmptyMethod:
   EnforcedStyle: expanded
 
+# Trailing commas in array literals helps having smaller diffs when we want to add additional parameters
```

Comment:
👍 

Totally out of topic and not really a Ruby friendly thing, but I do love how Elm does trailing commas

```elm
[ button [ onClick Decrement ] [ text "-" ]
, div [] [ text (String.fromInt model) ]
, button [ onClick Increment ] [ text "+" ]
]
```